.github/workflows/pre-commit.yml

```diff
@@ -0,0 +1,12 @@
+name: Linting
+
+on: [push]
```

Comment:
Will this still appear as a PR check? Do we need to include `pull_request` here?

Reply:
Oh, we might. Good catch